Acidanthera Updates December 2020 (#14)
dortania/dortania.github.io: _posts/2020-12-07-acidanthera-december.md

---
layout: post
title:  "Acidanthera Updates: December 2020"
date:   2020-12-07 15:00:00 -0700
categories: Hackintosh updates
---

* [A message from vit9696](#a-message-from-vit9696)
* [Dortania Updates](#dortania-updates)
* [Changelogs](#changelogs)

# A message from vit9696

{Inspirational message}

— Vit

# Dortania Updates

> This month has been a fairly quiet commit cycle for us on the guides, and thanks to this bit of peace we here at Dortania have been able to dedicate more time into ome smaller projects. The project we've been hard at work on this month has been our [OpenCore Legacy Patcher](https://github.com/dortania/Opencore-Legacy-Patcher). 

Reviewer: ```suggestion
This month has been a fairly quiet commit cycle for us on the guides, and thanks to this bit of peace we here at Dortania have been able to dedicate more time to some smaller projects. The project we've been hard at work on this month has been our [OpenCore Legacy Patcher](https://github.com/dortania/Opencore-Legacy-Patcher). 
```

The goal of this tool is simple:

* Allow Mac machines dropped by Apple to run newer versions of macOS
* And ensure all of this is kept in memory, allowing for all security features compared to other basic patchers

Because of this philosophy, we've gained a lot of features most patchers cannot compare to. This includes:

* Native OTA updates
* Full SIP, AMFI, Gatekeeper and other macOS secuirty features
* Full FileVault 2 support
* APFS Snapshotting
* Native IO80211 usage for many broadcom cards

> * Only current way to boot Arrendale, Lynnfield and Clarkdale Macs

Reviewer: ```suggestion
* Only current way to boot Arrandale, Lynnfield and Clarkdale Macs
```

  * ie. iMac11,1, iMac11,2, iMac11,3, MacBookPro6,1, MacBookPro6,2, MacBookPro6,3
 

> We hope that the community can also use our tools to the fullest pottential, and we want to give a big thanks to Acidanthera for the countless hours, days, months and years spent working on tools used in our project such as OpenCorePkg, Lilu, WhateverGreen, etc 

Reviewer: ```suggestion
We hope that the community can also use our tools to the fullest potential, and we want to give some big thanks to Acidanthera for the countless hours, days, months and years spent working on tools used in our project such as OpenCorePkg, Lilu, WhateverGreen, etc 
```
